src/datasets/constants.ts

```diff
@@ -6,7 +6,7 @@ export const SEARCH_ITEMS_PER_PAGE = 16;
  *
  * @link https://github.com/open-feature/.github/tree/main/templates/READMEs
  */
-export const features = [
+export const serverSideFeatures = [
```

Comment:
I was going to extend a shared list of features but I wanted better control over the ordering.